src/vs/languages/html/common/htmlTagSpecifications.ts

```diff
@@ -0,0 +1,295 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
I copied it from a [sibling file](https://github.com/Microsoft/vscode/blob/master/src/vs/languages/html/common/htmlTags.ts) since `gulp hygiene` failed the build.